examples/adb-lakehouse/variables.tf

```diff
@@ -0,0 +1,70 @@
+variable "location" {
+  type        = string
+  description = "(Required) The location for the resources in this module"
+}
+
+variable "spoke_resource_group_name" {
+  type        = string
+  description = "(Required) The name of the Resource Group to create"
+}
+
+variable "project_name" {
+  type        = string
+  description = "(Required) The name of the project associated with the infrastructure to be managed by Terraform"
+}
+
+variable "environment_name" {
+  type        = string
+  description = "(Required) The name of the project environment associated with the infrastructure to be managed by Terraform"
+}
+
+variable "vnet_name" {
+  type        = string
+  description = "Name of existing virtual network into which Databricks will be deployed"
+}
```

Comment:
README says that we create a vnet, but here we're talking about existing virtual network?